src/foam/blob/HashingOutputStream.java

```diff
@@ -0,0 +1,47 @@
+package foam.blob;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class HashingOutputStream
```

Comment:
Add some docs.

Reply:
Done. Also added missing license header to HashingOutputStream and InputStreamBlob